src/entities/item/model/useItemSubmit.ts

```
@@ -1,70 +1,62 @@
-            }
+    formData.append(
+        "request",
+        new Blob([JSON.stringify(request)], { type: "application/json" }),
+        "request.json"
+    );
 
-            const isEdit = !!itemId;
-            const url = isEdit ? `/api/v1/items/${itemId}` : "/api/v1/items/new";
-            const method = isEdit ? "patch" : "post";
+    if (reviewImages && reviewImages.length > 0) {
+        reviewImages.forEach((file) => formData.append("reviewImages", file));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)